ghDepsInstall.sh

```diff
@@ -1,5 +1,6 @@
 #! /bin/bash
 
 if [[ $(git config --get remote.origin.url) == *buildo/react-components.git* ]]; then
-  npm i ./ghDeps;
+  # use yarn if installed, or use npm instead
+  command -v yarn > /dev/null 2>&1 && yarn add ./ghDeps || npm i ./ghDeps
```

Comment:
as I understand it, `yarn add` is the equivalent of `npm i --save` (not just `npm i`). Aren't we adding more deps to the top-level package here (eg: `"ghDeps": "file://./ghDeps"`)? Or does it work differently in this case?